Artifacts/linux-syscoin/build_syscoin.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -e 
+
+date
+ps axjf
+
+
+if [ $1 = 'From_Source' ]; then
+#################################################################
+# Build Syscoin from source                                     #
+#################################################################
+NPROC=$(nproc)
+echo "nproc: $NPROC"
+#################################################################
+# Install all necessary packages for building Syscoin           #
+#################################################################
+sudo apt-get -y install git build-essential libtool autotools-dev autoconf pkg-config libssl-dev libevent-dev bsdmainutils libboost-all-dev libminiupnpc-dev libzmq3-dev
```

Comment:
To support inclusion in the public artifact repository, you need to support Debian (apt-get) and Fedora (yum) package managers.  Also, you will need to ensure your artifact supports the following Azure-based linux distros: Ubuntu, Fedora RedHat, CoreOS, CentOS, and SuSE.  Look at some of the other linux artifacts in the repository to see examples on how to do this.

Alternatively, host your own public-based artifact repository.